app/src/main/java/org/schabi/newpipe/player/datasource/YoutubeHttpDataSource.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Based on ExoPlayer's DefaultHttpDataSource, version 2.17.1.
+ * Based on ExoPlayer's DefaultHttpDataSource, version 2.18.0.
```

Comment:
I don't think there's any point in this change?

Reply:
See https://github.com/TeamNewPipe/NewPipe/pull/8536#issuecomment-1166390835. Anyway, this class will be removed soon in a future PR, which will use OkHttp as a network stack for playback and update the library on the fly.

Reply:
Thanks for pointing that out, I didn't notice the comment.